apps/frontend/src/lib/activity/index.ts

```diff
@@ -54,3 +54,15 @@ export const createTapActivity = (
     timestamp: new Date(),
   };
 };
+
+export const PSIActivityDataSchema = z.object({
+  connectionUsername: z.string(),
+});
+export type PSIActivityData = z.infer<typeof PSIActivityDataSchema>;
```

Comment:
Move to types?

Reply:
right now all activity types and schemas are defined here, and are only used in context of activity helpers. but in the future could be worth a refactor